app/views/layouts/_ga.html.erb

```diff
@@ -0,0 +1,12 @@
+<!-- Google Analytics -->
+<script type="text/javascript">
+    (function (i, s, o, g, r, a, m) {
+        i['GoogleAnalyticsObject'] = r; i[r] = i[r] || function () {
+            (i[r].q = i[r].q || []).push(arguments)
+        }, i[r].l = 1 * new Date(); a = s.createElement(o),
+            m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m)
+    })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
+    ga('create', 'UA-55037512-1', 'auto');
+    ga('send', 'pageview');
+</script>
+<!-- End Google Analytics -->
```

Comment:
Nice extraction and DRY'ing up here!

Reply:
Do we want to extract the analytics UA code into an env variable, so it doesn't fire in development environments?

Reply:
Or simply only render the partial in production environments is another option

Reply:
I think option #1 is best- make it an env var, so we can test analytics in staging (under a different UA code). We've also got the GA upgrade to do in next 9 months...